server/app/api/v1/charity/raffle/controller.js

```diff
@@ -93,31 +93,32 @@ const create = (req, res, next) => {
 const purchase = (req, res, next) => {
   const hasUserId = req.query.hasOwnProperty('userId');
   if (req.raffle.isActive) {
-    const findOrCreateUser = hasUserId
-      ? findByUserId
-      : createUser;
+    const findOrCreateUser = hasUserId ? findByUserId : createUser;
 
     findOrCreateUser()
       .then(addTickets)
         .all()
       .then(([raffle, user]) => {
-        const buyer = raffle.purchases.filter(
+        const { email, firstName, lastName } = user;
+        const { count } = raffle.purchases.filter(
           purchase => purchase.buyer === user._id
```

Comment:
Does this have an implicit `return`? I didn't know ES was going to do that. Is it for all fat-arrow functions or just one-liner fat-arrows?